bundler.d/assets.rb

```diff
@@ -1,21 +1,18 @@
 group :assets do
   gem 'ace-rails-ap', '~> 4.0.0'
-  gem 'sass-rails', '~> 3.2'
+  gem 'sass-rails', '~> 4.0'
   gem 'uglifier', '>= 1.0.3'
   gem 'execjs', '>= 1.4.0', '<2.5.0'
-  gem 'jquery-rails', '2.0.3'
+  gem 'jquery-rails', '~> 2.0.3'
```

Comment:
No worries, I'll keep the packages using the same.  We can fix http://projects.theforeman.org/issues/8705 another time.